products.js

```diff
@@ -0,0 +1,308 @@
+/**
+ * Using fathom to extract a product from its product page,
+ * where a 'product' is defined by the bundle of features that
+ * makes it identifiable.
+ *
+ * Features:
+ *    In Progress - Title, Image, Price
+*     Future - Reviews, Product ID?
+ *
+ * Testing:
+ *    50 test products in product_classification_test_data folder
+ *    All test sources are product detail pages (one product on this page)
```

Comment:
I doubt that's true for Amazon; I think what we mean is one "main" product on a page.